### ISSUE TITLE
Missing textures for several nether brick blocks

### ISSUE THREAD

**Issue by MobButcher** [ConsistencyPlus/ConsistencyPlus#56]
**Describe the bug**
Some textures are missing from the mod assets folder, and thus, from the game.

**Versions**
Minecraft version: 1.17.1
C+ version: 0.4.0
Fabric API version: 0.11.6

**Other mods**
```
  [✔️] antighost-1.17.1-fabric0.36.1-1.1.4
  [✔️] architectury-2.2.21-fabric
  [✔️] betterbeds-1.2.0
  [✔️] capes-1.2.0
  [✔️] cloth-api-2.0.54
  [✔️] cloth-config-5.0.34-fabric
  [✔️] consistency_plus-1.17.1-0.4.0
  [✔️] cullleaves-2.2.0
  [✔️] custom-fog-1.6.0
  [✔️] dynamic-fps-2.0.4
  [✔️] EntityCulling-Fabric-1.3.3
  [✔️] fabric-api-0.37.0+1.17
  [✔️] fabric-language-kotlin-1.6.2+kotlin.1.5.20
  [✔️] fabrishot-1.5.1
  [✔️] FallingTree-1.17.1-2.12.3+187-uk-UA
  [✔️] fast-chest-1.2+1.17
  [✔️] ferritecore-3.0.1-fabric
  [✔️] hydrogen-fabric-mc1.17.1-0.3
  [✔️] item-model-fix-1.0.2+1.17
  [✔️] lambdynamiclights-2.0.1+1.17
  [✔️] lithium-fabric-mc1.17.1-0.7.3
  [✔️] megane-5.1.2
  [✔️] modmenu-2.0.2
  [✔️] nofade-1.17-2.0.0
  [✔️] notenoughcrashes-3.3.1+1.17-fabric
  [✔️] slight-gui-modifications-2.0.1
  [✔️] smoothboot-fabric-1.16.5-1.6.0
  [✔️] sodium-extra-mc1.17.1-0.3.3
  [✔️] sodium-fabric-mc1.17.1-0.3.0+build.5
  [✔️] WI-Zoom-1.3-MC1.17.1
  [✔️] wthit-fabric-3.8.1
  [✔️] yosbr-0.1.1
```

**Logs**
```
[22:50:08] [Worker-Main-3/ERROR]: Using missing texture, unable to load consistency_plus:textures/block/red_nether_brick_pillar.png : java.io.FileNotFoundException: consistency_plus:textures/block/red_nether_brick_pillar.png
[22:50:08] [Worker-Main-3/ERROR]: Using missing texture, unable to load consistency_plus:textures/block/chiseled_blue_nether_bricks.png : java.io.FileNotFoundException: consistency_plus:textures/block/chiseled_blue_nether_bricks.png
[22:50:11] [Worker-Main-3/ERROR]: Using missing texture, unable to load consistency_plus:textures/block/blue_nether_brick_pillar_top.png : java.io.FileNotFoundException: consistency_plus:textures/block/blue_nether_brick_pillar_top.png
[22:50:12] [Worker-Main-3/ERROR]: Using missing texture, unable to load consistency_plus:textures/block/blue_nether_brick_pillar.png : java.io.FileNotFoundException: consistency_plus:textures/block/blue_nether_brick_pillar.png
[22:50:12] [Worker-Main-3/ERROR]: Using missing texture, unable to load consistency_plus:textures/block/carved_red_nether_bricks.png : java.io.FileNotFoundException: consistency_plus:textures/block/carved_red_nether_bricks.png
[22:50:12] [Worker-Main-3/ERROR]: Using missing texture, unable to load consistency_plus:textures/block/nether_brick_pillar_top.png : java.io.FileNotFoundException: consistency_plus:textures/block/nether_brick_pillar_top.png
[22:50:12] [Worker-Main-3/ERROR]: Using missing texture, unable to load consistency_plus:textures/block/carved_blue_nether_bricks.png : java.io.FileNotFoundException: consistency_plus:textures/block/carved_blue_nether_bricks.png
[22:50:13] [Worker-Main-3/ERROR]: Using missing texture, unable to load consistency_plus:textures/block/red_nether_brick_pillar_top.png : java.io.FileNotFoundException: consistency_plus:textures/block/red_nether_brick_pillar_top.png
[22:50:13] [Worker-Main-3/ERROR]: Using missing texture, unable to load consistency_plus:textures/block/chiseled_red_nether_bricks.png : java.io.FileNotFoundException: consistency_plus:textures/block/chiseled_red_nether_bricks.png
[22:50:15] [Worker-Main-3/ERROR]: Using missing texture, unable to load consistency_plus:textures/block/nether_brick_pillar.png : java.io.FileNotFoundException: consistency_plus:textures/block/nether_brick_pillar.png
[22:50:15] [Worker-Main-3/ERROR]: Using missing texture, unable to load consistency_plus:textures/block/chiseled_nether_bricks.png : java.io.FileNotFoundException: consistency_plus:textures/block/chiseled_nether_bricks.png
```

**To Reproduce**
1. Launch the game.
2. Take one of the blocks listed in the log.
3. Observe.

**Expected behavior**
Blocks should have textures.

**Comment by Cart3r1234** [comment]
This is intentional; these blocks are being changed sometime soon (0.4.1/1.0) so I removed their textures early.  They should be unobtainable in survival.  I'll leave this issue open until the change actually occurs though.

**Comment by Siuolthepic** [comment]
Wait, was this shipped in 0.4.1?

**Comment by Cart3r1234** [comment]
This should also be in 0.4.1rc1
